app/views/site/contribute.erb

```diff
@@ -0,0 +1,130 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Contributing Guide</h1>
+
+    <p class="lead">
+      <strong>So you want to contribute to exercism?</strong><br />
+      That is great news! We welcome all sort of contributions and will do everything we can to help you get started.
+    </p>
+
+    <p>
+      <strong>Exercism is a project comprised of several repositories and these can be divided basically in:</strong>
+
+      <ol>
+        <li><strong>The website</strong>: Its here that we can improve exercism features or layout.</li>
```

Comment:
This one is different from all of the others. All of the following bullet points explain what the thing does, whereas this one says how to contribute. How about saying something like:

_Browse and review solutions, and have conversations._